vendor/assets/javascripts/gemini/gemini_upload.js

```diff
@@ -38,7 +38,7 @@ $.fn.extend({
     successAction = data.policy_document.conditions[3].success_action_status;
     base64Policy = data.policy_encoded;
     signature = data.signature;
-    s3Key = options.s3Key;
+    s3Key = data.credentials;
```

Comment:
This was referring to the option specified by the client: https://github.com/artsy/volt/blob/87fcb23b0d7b844fd3df71e9917065abec21dd6d/app/assets/javascripts/gemini_uploads.js.coffee#L99 , which we now realized is both redundant _and_ brittle.